include/common_structs.h

```diff
@@ -240,31 +239,31 @@ typedef struct {
     /* 0x0198 */ f32 unk_198;
     /* 0x019C */ f32 unk_19C;
     /* 0x01A0 */ f32 unk_1A0;
-    /* 0x01A4 */ s8 unk_1A4;
+    /* 0x01A4 */ s8 frontLeftTireTerrain;
```

Comment:
I do lean towards `surface` because this term is used in Overkart 64. This allows the code-base to use similar terminology as standardized by the Mario Kart modders.

Reply:
Also, it's spelled tyre, no?

Reply:
Surface sounds good. And tire/tyre is American vs British spelling. I don't have any attachment to either though

Reply:
Hmm... I think tyre is the global spelling as I'm not British and we spell it this way. I'm wondering if someone might confuse the symbol for deterioration.